grails-app/controllers/grails/plugin/sentry/SentryTracingInterceptor.groovy

```diff
@@ -0,0 +1,22 @@
+package grails.plugin.sentry
+
+import org.springframework.web.servlet.HandlerMapping
+
+class SentryTracingInterceptor {
```

Comment:
This interceptor is the same we have in our grails app. Would we be able to remove it in the grails-app ?

Reply:
This is default, that can be overwritten. It's still useful for performance monitoring working out of the box. Much better approach would be [providing custom TransactionNameProvider](https://github.com/getsentry/sentry-java/pull/1861) bean, but that is possible only in sentry-java v6 which is still beta.